Decred Wallet/Features/Send/ConfirmToSendFundViewController.swift

```diff
@@ -91,20 +91,20 @@ class ConfirmToSendFundViewController: UIViewController, UITextFieldDelegate {
         if let exchangeRate = self.exchangeRate {
             let sendingAmountUsd = self.unsignedTxSummary.dcrAmount.multiplying(by: exchangeRate)
             let sendingAmountUsdText = Utils.styleAttributedString(
-                " (\(sendingAmountUsd.round(4).formattedWithSeparator))",
+                " ($\(sendingAmountUsd.round(4).formattedWithSeparator) USD)",
```

Comment:
```suggestion
                " ($\(sendingAmountUsd.round(2).formattedWithSeparator))",
```
#645 says to use 4 decimal places for fees only, 2 decimal places otherwise.